pkg/machines/getOSList.py

```diff
@@ -44,6 +53,39 @@
         storage = minimumResources.get_nth(0).get_storage()
         if storage != -1:
             osObj['minimumResources']['storage'] = storage
+
+    osObj['profiles'] = []
+    osInstallScripts = os.get_install_script_list()
+    osObj['profiles'].extend(_getInstallScriptProfile(osInstallScripts))
+
+    osObj['unattendedInstallable'] = False
+    installer_scripts = os.get_install_script_list()
+    if installer_scripts.get_length() > 0:
+        osObj['unattendedInstallable'] = True
+
+    osObj['medias'] = {}
+    osMedias = os.get_media_list()
+    for j in range(osMedias.get_length()):
+        media = osMedias.get_nth(j)
+        mediaId = media.get_id()
+
+        osObj['medias'][mediaId] = {}
+        osObj['medias'][mediaId]['unattendedInstallable'] = False
+        osObj['medias'][mediaId]['profiles'] = []
+
+        if (osObj['unattendedInstallable'] and
+           hasattr(media, 'supports_installer_script')):
+            supports = media.supports_installer_script()
```

Comment:
as well, querying twice the same

Reply:
Where? I can't see what you mean.
If you mean the hasattr and the function call bellow they are both needed, otherwise the script will fail when running it with libosinfo version which does not have the function in line 78.

Reply:
Done.